### DISCUSSION TITLE
Amazon Linux 2022 for workspaces

### DISCUSSION THREAD

**Discussion by msilveuseagle6** [amazonlinux/amazon-linux-2022#261] [Q&A]
Hi all.  My company is hoping to use Amazon Linux 2022 in our workspace environments for some of its modern feature.  Is there any timeline for making this available and if so what is the timeline?

**Comment by stewartsmith** [comment]
We don't currently have a timeline that we are sharing publicly.

**Comment by owengo** [comment]
We are also interested, is it possible to know if at least it's in some pipeline? We would rather not move to ubuntu workspace.
